FreydCategoriesForCAP/tst/050_load_with_suggested_dependencies.tst

```diff
@@ -7,8 +7,12 @@ gap> package_loading_info_level := InfoLevel( InfoPackageLoading );;
 gap> SetInfoLevel( InfoPackageLoading, PACKAGE_ERROR );;
 gap> LoadPackage( "FreydCategoriesForCAP", false );
 true
+gap> LoadPackage( "FinSetsForCAP", false );
```

Comment:
The `CAP_project`-ci does not include `FinSetsForCAP`. Please reset this file to the original state.